justfile

```diff
@@ -0,0 +1,38 @@
+# Default recipe that lists all available commands
```

Comment:
Why didn't you use `Makefile`? Do we need to install `just`?

Reply:
@hramezani `Justfile` isn’t as much of a “standard” as `Makefile`, but it’s now fairly common in open-source projects as a task runner. 
for `django-auditlog` porject needs (i18n, test, install), just’s command-runner model felt like a better fit. 
Its simpler syntax also makes commands easier to maintain, and it works consistently across Windows, macOS, and Linux.

Reply:
>  also we need to install just to use it. I prefer Makefile because it is common and can cover our use case.

@hramezani makes sense. I picked Justfile since I had seen and used it frequently in past projects, but I agree Makefile is the right fit here since it’s dependency-free and already the Jazzband convention.